modules/installation-configuring-shiftstack-single-ipv6.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-configuring-shiftstack-single-ipv6_{context}"]
+= Configuring a cluster with single-stack IPv6 networking
+
+You can create a single-stack IPv6 cluster on {rh-openstack-first} after you configure your {rh-openstack} deployment.
+
+IMPORTANT: You cannot convert a dual-stack cluster into a single-stack IPv6 cluster.
+
+.Prerequisites
+
+* Your {rh-openstack} deployment has an existing network with a DHCPv6-stateful IPv6 subnet to use as the machine network.
+* DNS is configured for the existing IPv6 subnet.
+* The IPv6 subnet is added to a {rh-openstack} router, and the router is configured to send router advertisements (RAs).
+* You added any additional IPv6 subnets that are used in the cluster to an {rh-openstack} router to enable router advertisements.
++
+NOTE: Using an IPv6 SLAAC subnet is not supported because defined `dns_nameservers` servers are not enforced by {rh-openstack} Neutron.
```

Comment:
```suggestion
NOTE: Using an IPv6 SLAAC subnet is not supported because any `dns_nameservers` defined are not enforced by {rh-openstack} Neutron.
```

Reply:
```suggestion
NOTE: Using an IPv6 SLAAC subnet is not supported because any `dns_nameservers` addresses are not enforced by {rh-openstack} Neutron.
```

@MaysaMacedo Our style guide requires a noun after something like `dns_nameservers`. What do you think?